conda-forge.yml

```
@@ -1,3 +1,5 @@
+azure:
+  upload_packages: False
```

Sorry copy-pasted these lines. This should be `True`

```suggestion
  upload_packages: True
```